package.json

```diff
@@ -2,20 +2,23 @@
   "name": "isomercms-frontend",
   "version": "0.16.1",
   "private": true,
+  "engines": {
+    "node": ">=16.0.0"
+  },
   "dependencies": {
     "@atlaskit/tree": "^7.0.2",
```

Comment:
@alexanderleegs @kishore03109 @seaerchin where are we using this package? Couldn't find it in the code

Reply:
do you mean `@atlastkit/tree`? i also have no clue so feel free to remove kek